products/appengine/api.yaml

```diff
@@ -440,3 +440,39 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'UrlDispatchRule'
+    description: |
+      Rules to match an HTTP request and dispatch that request to a service.
+    base_url: ''
+    create_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    create_verb: :PATCH
+    delete_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    delete_verb: :DELETE
```

Comment:
Will need to be `:PATCH` as well - essentially all of these operations are patching a list of dispatch rules on the App resource in GCP to add/update/remove items in the list

Reply:
changed delete to PATCH